elf/Kconfig

```diff
@@ -66,3 +66,10 @@ config MPU_TEST_APP_ADDR
 	---help---
 		Test MPU access protection to this application address
 endif
+
+config PREAPP_FROM_ELF
+	bool "Start preapp from ELF"
+	default y
+	---help---
```

Comment:
I think conditional dependency on SYSTEM_PREAPP_INIT is required since both cannot be used simultaneously.

Reply:
It is not required because both need to be active in our case. PREAPP must be enabled and FROM_ELF also needs to be enabled to launch preapp from elf. In other cases, PREAPP will be enabled, but FROM_ELF is not enabled, so that preapp will be launched from system boot.